autokeras/blocks/basic.py

```diff
@@ -789,11 +789,11 @@ class BertBlock(block_module.Block):
     ```python
         # Using the Transformer Block with AutoModel.
         import autokeras as ak
-        from autokeras import BERTBlock
+        from autokeras.blocks import BertBlock
```

Comment:
Please remove the .blocks since it can be directly imported from autokeras with `from autokeras import BertBlock`.
https://github.com/keras-team/autokeras/blob/master/autokeras/__init__.py#L16

Reply:
Done! Sorry about that -- I should have caught that.